docs/android.md

```diff
@@ -2,8 +2,8 @@
 This document explains how to include Unity as a Library into standard Android application through Activity. You can read more about [Unity as a Library](https://docs.unity3d.com/2019.3/Documentation/Manual/UnityasaLibrary.html).
 
 **Requirements:**
-- Android Studio 3.4.2+
-- Unity version 2019.3.0b4+
+- Android Studio Bumblebee (2021.1.1) or later
+- Unity version 2022.2.0a18 or later
 
```

Comment:
For anyone looking for older unity version solution, i would leave a note like:
[NOTE] For previous Unity version 2019.3.0b4+ [click here](**url**) 

please change click here to correct url

Reply:
Good notice. I was planning to do this - I even created a separate branch for an old version before I started an update, but then I forgot to mention it here :D Done in https://github.com/Unity-Technologies/uaal-example/pull/70/commits/fb15cab3b1ca6358f993d81f84ad877d5f02c6b1